pyartifactory/models/user.py

```diff
@@ -47,7 +47,7 @@ class User(BaseUserModel):
 class UserResponse(BaseUserModel):
     """Models a user response."""
 
-    email: EmailStr
+    email: Optional[EmailStr] = None

```

Comment:
When looking at the [Artifactory API](https://jfrog.com/help/r/jfrog-rest-apis/security-configuration-json?section=UUID-6d36b713-8bae-89cd-087a-b4394f19049c_id_SecurityConfigurationJSON-applicationvndorgjfrogartifactorysecurityUserjson), I can see that the email param is mandatory

Reply:
Never mind, there was an issue with our enterprise settings that wasn't filling the email field. I fixed it, closing this PR.